src/EpplexProvider.ts

```diff
@@ -397,12 +412,22 @@ class EpplexProvider {
                 gameConfig: getGameConfigAccount(),
                 updateAuthority: programDelegate,
                 token22Program: TOKEN_2022_PROGRAM_ID,
-                tokenProgram: TOKEN_PROGRAM_ID
+                tokenProgram: TOKEN_PROGRAM_ID,
             })
             .transaction();
 
         return tokenBurnTx;
     }
+
+    async getGameConfig(): Promise<GameConfig> {
+        const gameConfig = getGameConfigAccount();
```

Comment:
we have examples on how to derive PDAs inside the constants folder

Reply:
this is for fetching the game config account

Reply:
right, was referring to the fact that we don't need a separate pda.ts, could follow how we derive the other PDAs

Reply:
sounds good, let me make the changes.